### PR TITLE
updates made to tag cloud

### PR DESCRIPTION
**Pull request by escottalexander**
I like the changes. Their is more work to do on this though. If anyone wants to continue the tag cloud project, you can find most of the settings are easy to play with at the end of the jekyll/assets/js/jquery.tagcanvas.js file. (#50)

### CODE REVIEW

**Comment by andy5995** [comment]
omg @escottalexander , that is so cool!

screenshot_2018-01-16_10-49-20

I haven't ever seen a tag cloud like that before :)

I find it a little difficult to use though. It took me a couple minutes to figure how to find and click on one that I wanted. I had to center a link before I could click on it. It's very sensitive to movement and the link "spins" away if I don't move the mouse a certain way. Is this the same behavior in your web browser? (I'm using Firefox on Debian 9)

And for some reason, the font on some links are big that shouldn't be. Like the tag "hulpverleners-van-korrelatie/" shows (28) next to it. It should only be (1).

Thank you for this, and I hate to say this after the work you put into it, but unless the issues I mentioned can be fixed, we won't be able to use it. I'll keep this open though, of course.

It is really cool though!

**Comment by escottalexander** [comment]
I have played with it some more. Let me know if you think its any better. I know it still needs some work. I am not sure what is causing the incorrect quantity labels on the tags. I am thinking more and more we might just need to scrap method and go for a traditional tag cloud. This would be good if there was 50 tags but I think there are to many to try to visualize them like this. Any thoughts?

File: jekyll/_layouts/tag_cloud.html [hunk 1]
```diff
@@ -4,14 +4,23 @@
...
+    {% assign tags = site.tags | sort %}
+    {% for tag in tags %}
+    <li class="site-tag">
       <a href="/tag/{{ tag | first | slugify }}/"
-          style="font-size: {{ tag | last | size  |  times: 4 | plus: 80  }}%">
-              {{ tag[0] | replace:'-', ' ' }} ({{ tag | last | size }})
-      </a>
-    </span>
+      style="font-size: {{ tag | last | size  |  times: 2 | plus: 80  }}%">
+      {{ tag[0] | replace:'-', ' ' }} ({{ tag | first | size }})
```
Comment (andy5995) [inline jekyll/_layouts/tag_cloud.html]: When I change "first" to "last" in this line, the counts show correctly. I think it just has to match with the "last" in the line above it. Is there a reason to tried changing it, Elliot?

**Comment by andy5995** [comment]
>I am thinking more and more we might just need to scrap method and go for a traditional tag cloud. This would be good if there was 50 tags but I think there are to many to try to visualize them like this. Any thoughts?

This one's a lot easier to use, but I'm inclined to agree.

But I bet this was fun to work on. :) Was this something new your were learning/trying?

File: jekyll/_layouts/tag_cloud.html (hunk 1 above)
Comment (escottalexander) [reply jekyll/_layouts/tag_cloud.html]: I was just toying with it and forgot how it was originally. I am positive that both lines were different though.

**Comment by escottalexander** [comment]
It has been fun to play with and maybe I will find a use for this neat concept somewhere else in the future. How is the best way to nullify my changes? Deleting the branch? I will start from scratch with a more typical tag cloud.

File: jekyll/_layouts/tag_cloud.html (hunk 1 above)
Comment (andy5995) [reply jekyll/_layouts/tag_cloud.html]: I will have to look at this diff when I'm not tired. :)

https://github.com/mhwkb/mhwkb.github.io/pull/121/files

Near the top of every PR, the "files changed" tab will show you the differences between the originals and the files you changed.

**Comment by andy5995** [comment]
>How is the best way to nullify my changes? Deleting the branch? I will start from scratch with a more typical tag cloud.

You or I can close the PR. You don't have to delete the branch, just remember to switch back to master and sync it before starting with something new.

have a good night! And.. how'd your interview go?

**Comment by escottalexander** [comment]
The job interview with the Ruby question? I was told that they are pursuing more experienced candidates. But a friend of mine has helped me get an on-site interview with a really great company in Indiana. (I live in Virginia) The interview is generally 5 hours long and includes a whiteboard so I am pretty nervous. Lol Any interview advice?

**Comment by andy5995** [comment]
@escottalexander That sounds like a really good opportunity.

The best thing I could think of for to prepare is to find people with whom you can have practice interviews. Sometimes states or counties offer that for free through job service programs. One job service center I went to had an A/V section where I could watch educational videos.

One question that employers often ask is, "What are your weaknesses?" Now.. you may think a great answer would be, "I'm a perfectionist", however, that's too commonly used.. So get creative with a prepared answer to that question.

My sister approaches interviews trying to use the mindset.. "I'm interviewing *them*".

Basically the more relaxed and confident you are going in, the better off you'll be. Remember to breathe, and remind yourself that's it's only 1 interview. And if you don't get hired, be grateful for the practice.

Also.. ask @erichoog . I think he would have some good insight in this area. ;)

**Comment by escottalexander** [comment]
Thanks for the thoughts. @erichoog Any additional advice?

**Comment by erichoog** [comment]
@escottalexander I honestly still haven't mastered the interview.  I get super nervous and forget things and my communication goes to crap lol the things @andy5995 has said are helpful for sure!  There are some github repos that have interview help for languages like javascript but i can't remember the name off the top of my head. I'll post it here if I find it.
